src/event_handlers/hubV2.ts

```diff
@@ -273,23 +261,14 @@ NameRegistry.UpdateMetadataDigest.handler(async ({ event, context }) => {
 
 HubV2.StreamCompleted.handlerWithLoader({
   loader: async ({ event, context }) => {
-    let transfers = await context.Transfer.getWhere.transactionHash.eq(
+    let transfers = await context.Transfer.getWhere.transaction_id.eq(
       event.transaction.hash
     );
 
-    return {
-      transfers,
-      demurrageFrom: transfers.filter(
-        (t) => t.to === zeroAddress && t.from === event.params.from
-      ),
-      demurrageTo: transfers.filter(
-        (t) => t.to === zeroAddress && t.from === event.params.to
-      ),
-      metriFee: transfers.filter((t) => t.transferType === "MetriFee"),
-    };
+    return { transfers };
   },
   handler: async ({ event, context, loaderReturn }) => {
-    const { transfers, demurrageFrom, demurrageTo, metriFee } = loaderReturn;
```

Comment:
at first i thought that this info was lost, but if i understand correctly the Demurrage transfer type will be linked to the top transaction 👍

Reply:
yes, both demurrage and metri fee. And all future fees